configs/siklu_80X0_defconfig

```diff
@@ -2,6 +2,9 @@ BR2_arm=y
 BR2_cortex_a7=y
 BR2_ARM_FPU_VFPV4=y
 BR2_SVN="svn"
+BR2_DL_DIR="${CACHE_ROOT_DIR}/cache/siklu/portf/8020/buildroot-dl"
```

Comment:
this is not only for 8020

Reply:
The directory name needs to be a name for all projects that share the same version of:
1. Buildroot
2. Toolchain
That is, we should not use it for MH. So what should we call this directory instead of "8020"?

Reply:
> The directory name needs to be a name for all projects that share the same version of:
> 
> 1. Buildroot
> 2. Toolchain
>    That is, we should not use it for MH. So what should we call this directory instead of "8020"?

Maybe 80XX

Reply:
Ok